modules/webgl/package.json

```diff
@@ -41,7 +41,7 @@
     "@luma.gl/constants": "7.0.0-alpha.25",
     "@luma.gl/webgl-state-tracker": "7.0.0-alpha.25",
     "@luma.gl/webgl2-polyfill": "7.0.0-alpha.25",
-    "probe.gl": "3.0.0-alpha.6",
+    "probe.gl": "3.0.0-alpha.7",
```

Comment:
Are we making more breaking changes to probe.gl? If not let's publish 3.0.0 as production release and get the dependencies updated to "^3.0.0" (with leading caret).